content/en/database_monitoring/setup_oracle/_index.md

```diff
@@ -24,26 +24,93 @@ The features described on this page are in private beta.
 The following deployment configurations, components, and features are not supported: Oracle Autonomous Database, ASM, and Data Guard.
 
 
-## Supported Agent versions
+## Latest Agent version
 
-The following custom build is supported: `7.44.1~dbm~oracle~beta~0.28`.
+The following beta build contains implemented Oracle DBM features: `7.44.1~dbm~oracle~beta~0.28`.
 
 ## Prerequisites
 
-1. **Install the Oracle integration**: On the [**Integrations**][1] page in Datadog, install the [Oracle integration][2] for your organization. This will install an Oracle dashboard in your account which can be used to monitor the performance of your Oracle databases.
-2. **Upgrade the Oracle integration in your Agent**: You can skip this step if this is your first time installing the Oracle integration. If you've previously installed the Oracle integration, migrate the legacy configuration from the `conf.d/oracle.d/` directory to the new integration path located in the `conf.d/oracle-dbm.d/` directory.
+### Install the Oracle integration
+
+On the [**Integrations**][1] page in Datadog, install the [Oracle integration][2] for your organization. This will install an [Oracle dashboard][5] in your account which can be used to monitor the performance of your Oracle databases.
+
+### Upgrade the Oracle integration in your Agent
+
+You can skip this step if this is your first time installing the Oracle integration. If you already installed the Oracle integration, migrate the legacy configuration from the `conf.d/oracle.d/` directory to the new integration path located in the `conf.d/oracle-dbm.d/` directory.
 
 Use the following command to migrate the Oracle integration from the legacy integration to the new one:
 
-```bash
-mv /etc/datadog-agent/conf.d/oracle.d/conf.yaml /etc/datadog-agent/conf.d/oracle-dbm.d/conf.yaml
+```shell
+cp /etc/datadog-agent/conf.d/oracle.d/conf.yaml /etc/datadog-agent/conf.d/oracle-dbm.d/conf.yaml
+```
+
+Deactivate the legacy integration:
+
+```shell
+mv /etc/datadog-agent/conf.d/oracle.d/conf.yaml /etc/datadog-agent/conf.d/oracle.d/conf.yaml
+```
+
+Deactivating the legacy integration prevents sending the system metrics twice.
+
+Since the Agent doesn't require an external Oracle client, remove the `jdbc_driver_path` configuration parameter from the new parameter file `/etc/datadog-agent/conf.d/oracle-dbm.d/conf.yaml`. 
+
+### Install Agent
```

Comment:
I would change this to "Installing a custom Agent build" or something like that

Reply:
Actually, "Install Agent" covert both install options: official release and custom build.

Reply:
ah ok, sorry this is nitpicky, but can we do "Install the Agent" then?

Reply:
I'm trying to maintain as much consistency with our other docs as possible: https://docs.datadoghq.com/database_monitoring/setup_mysql/selfhosted/?tab=mysql56